RPIPs/RPIP-core-dev.md

```diff
@@ -0,0 +1,52 @@
+---
+rpip: TBC 
+title: Protocol Development
+description: Determines the process around core protocol development and funding
+author: Darren Langley (@langers)
+discussions-to: https://dao.rocketpool.net/t/dev-team-funding/2038
+status: Draft
+type: Meta
+created: 2023-10-31
+---
+
+## Abstract
+To ensure the long-term sustainability and continued innovation of the Rocket Pool protocol, this document lays out the high-level roles, responsibilities, processes, and funding for core protocol development.
+
+## Related RPIPs
+- [RPIP-10: pDAO Budget Allocation](https://rpips.rocketpool.net/RPIPs/RPIP-10)
+- [RPIP-24: oDAO Charter](https://rpips.rocketpool.net/RPIPs/RPIP-24)
+- [RPIP-25: RPL Inflation Allocation](https://rpips.rocketpool.net/RPIPs/RPIP-25)
+
+## Specification
+
+### Prioritisation
+
+- Every 12 months, the Core Protocol Team MUST facilitate a prioritisation discussion with the Rocket Pool community.
+- pDAO SHOULD engage in the prioritisation discussion to ensure development team prioritises are aligned with pDAO expectations.
+- The Core Protocol Team MUST summarise findings of prioritisation discussion and communicate them to the pDAO.
+
+### Roadmap
+
+- The Core Protocol Team MUST define an indicative protocol roadmap or release schedule based on the prioritisation findings.
+- The Core Protocol Team MUST provide quarterly updates on the roadmap.
+- The protocol roadmap is expected to evolve over time, the Core Protocol Team SHOULD communicate these changes in the quarterly updates. 
+
+### Governance
+
+- The Core Protocol Team MUST draft Rocket Pool Improvement Proposals (RPIPs) for new protocol features.
+- pDAO SHOULD engage in discussion and co-development of the new protocol features.
+- pDAO MUST vote the inclusion of new protocol features.
```

Comment:
This reads a bit weird, like all proposed features must pass, almost.

Maybe: `In order for new protocol features to be included, they MUST be approved by a pDAO vote.`